src/components/question.cjsx

```diff
@@ -57,50 +68,47 @@ module.exports = React.createClass
         removeAnswer={@removeAnswer}
         changeAnswer={@changeAnswer}/>)
 
-    <div>
+    <div className="question">
+
+      {if removeQuestion
+        <div>
+          {if canMoveRight
```

Comment:
Odd, I would have expected `canMoveLeft` to appear first in the DOM (but I don't have a screenshot to look at for reference)